From b75fd7175fc13c7354c9d232bd3812feab8d2ff9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Morten=20Johan=20S=C3=B8rvig?= <morten.sorvig@qt.io>
Date: Fri, 30 Oct 2020 14:04:29 +0100
Subject: [PATCH 07/15] wasm: remove some incorrect information

Thread support is off by default, but can be enabled
with -feature-thread.

Change-Id: I433534389f6c3cba2407db471491af29a1e19465
Reviewed-by: Paul Wicking <paul.wicking@qt.io>
---
 doc/src/platforms/webasm.qdoc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/src/platforms/webasm.qdoc b/doc/src/platforms/webasm.qdoc
index 918d2c23..586d35de 100644
--- a/doc/src/platforms/webasm.qdoc
+++ b/doc/src/platforms/webasm.qdoc
@@ -137,7 +137,8 @@ Build required modules:
     make module-qtbase module-qtdeclarative [other modules]
 \endcode
 
-The Qt build is a static build, and does not support threads.
+The Qt build is a static build, and does not support threads by default. Configure with  \c{-feature-thread}
+to enable thread support.
 
 \section3 Build and run your application
 
@@ -196,8 +197,7 @@ You can test-run your application in the following way:
 
 \section1 Limitations
 
-The most important limitation is that the Qt build is static,
-and does not support threads.
+The most important limitation is that the Qt build is static.
 
 For further info, refer to \l{qtwebassembly-platform-notes}
 
-- 
2.31.1

